cypress/integration/Stepper._spec.js

```diff
@@ -0,0 +1,227 @@
+/* This Source Code Form is subject to the terms of the Creative Commons
+ * License Chooser
+*/
+
+// test the interpolated library
+
+describe('Stpper.vue', () => {
+    it('renders correctly', () => {
+        cy.visit('/')
+    })
+
+    describe('Checks the user know the licence needed', () => {
+        it('Checks whether the CC0 license is selected in the dropdown', () => {
+        cy.get('.step-0')
+            .find('[value="yes"]')
+            .check()
+        cy.get('.next-button').click()
+        cy.get('select').select('CC0 1.0')
+        cy.get('.recommended-card').should('be.visible').contains('CC0 1.0 Universal')
+        cy.get('.next-button').click()
+        cy.get('.v-checkbox:nth-child(1) > input').check()
+        cy.get('.v-checkbox:nth-child(2) > input').check()
+        cy.get('.next-button').click()
+        cy.get('.license-use-card').should('be.visible').contains(' CC0 1.0 ')
+        })
+
+        it('Checks whether CC BY-NC-ND is selected in the dropdown', () => {
+            cy.visit('/')
+            cy.get('.step-0')
+            .find('[value="yes"]')
+            .check()
+            cy.get('.next-button').click()
+            cy.get('select').select('CC BY-NC-ND 4.0')
+            cy.get('.recommended-card').should('be.visible').contains('CC BY-NC-ND 4.0')
```

Comment:
Can you add the check for `After the user clicks 'Next', the Attribution details step opens (i.e., there is no Copyright Waiver step, only 3 steps total`